packages.json

```diff
@@ -2697,6 +2711,7 @@
   "https://github.com/ibm-cloud-security/appid-serversdk-swift.git",
   "https://github.com/ibm-cloud-security/swift-jwk-to-pem.git",
   "https://github.com/IBM-Swift/alert-notification-sdk.git",
+  "https://github.com/IBM-Swift/SwiftyJSON.git",
```

Comment:
Also dropping this one for now, due to https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2809